olive/model/__init__.py

```diff
@@ -509,6 +527,16 @@ def script_dir(self) -> str:
     def model_script(self) -> str:
         return self.get_local_resource("model_script")
 
+    def extend_model_attributes_from_hf_config(self) -> Dict[str, Any]:
+        hf_model_config = {}
```

Comment:
ditto here. The priority logic should be put another place.

Reply:
Let us discuss offline.